wikiHandler.js

```diff
@@ -48,11 +53,14 @@ async function fetchAndFormatAuctionData(auction_user, auction_contents, server)
         }
         fields.push(field);
     })
+
     return new Discord.MessageEmbed()
         .setColor(SERVER_COLOR[server])
-        .setTitle(`${auction_user} (${auction_mode})`)
+        .setTitle(`**${auction_mode}**`)
         .setDescription(`\`\`\`${auction_contents}\`\`\``)
-        .addFields(fields);
+        .addFields(fields)
+        .setFooter(`${auction_user} on Project 1999 ${utils.formatCapitalCase(server)}`)
```

Comment:
Just actually noticed that the Auctioneer is only in the footer now -- I guess this is fine, though I kinda liked having it more visible I guess? And it's not like we're short on the space in that line, since it is a whole line by itself, plenty of space. You really prefer not to put the name there?

Reply:
I did a lot of tossing and turning over this, but I do think it's best this way.  In my opinion, having just the auction type in the headline is cleaner and makes it easier to quickly look a stream and pick out which is message is.  The timestamp at the bottom is necessary for mobile users who can't inspect the time for each message, so adding the auctioneer here isn't wasted space either.  My thoughts were that the only time a user cares who the auctioneer is is when they actually want to buy/sell the item, which is rare compared to wanting to know the auction type, items, prices, and other info.  I also think having the auctioneer by the timestamp makes sense together..."Rabitz on Project 1999 Blue | 10:27am" is all related info in my eyes.  Can create a poll for it too, I hear your point that it's easier to read in the title- I could be wrong that users want a less cluttered experience.